code/datums/weather/weather.dm

```diff
@@ -106,6 +106,8 @@
 
 /datum/weather/proc/can_weather_act(mob/living/L) //Can this weather impact a mob?
 	var/turf/mob_turf = get_turf(L)
+	if(!istype(L))
```

Comment:
What was happening here?

Reply:
This was runtiming pretty hard, but I can't remember. Either there were nulls there or a non-mob was passed.

Reply:
what kind of runtime? 

If it's null, then doing an `if(!L)` check would be better, to avoid doing an `istype` check on things.

Reply:
I can't really remember anymore at this point whether or not it was just a null. I don't think this is too much of a performance impact to warrant going back and finding the runtime though.

Reply:
Here it is:
```[2019-04-19T23:39:51] Runtime in weather.dm,111: undefined variable /mob/dview/var/weather_immunities
  proc name: can weather act (/datum/weather/proc/can_weather_act)
  src: ash storm (/datum/weather/ash_storm)
  call stack:
  ash storm (/datum/weather/ash_storm): can weather act(the dview (/mob/dview))
  Weather (/datum/controller/subsystem/weather): fire(0)
  Weather (/datum/controller/subsystem/weather): ignite(0)
  Master (/datum/controller/master): RunQueue()
  Master (/datum/controller/master): Loop()
  Master (/datum/controller/master): StartProcessing(0)```

Reply:
ohh, that's the hidden `dview` mob that we generate to test for certain visibilty checks
This test makes sense in that context, although I wonder if there's other ways to resolve the matter